src/utils/iso3_utils.py

```diff
@@ -144,6 +145,19 @@ def determine_max_adm_level(row):
         return min(1, row["src_lvl"])
 
 
+def load_coverage():
+    pipelines = ["seas5", "era5", "imerg", "floodscan"]
+    coverage = {}
+
+    for dataset in pipelines:
+        config = load_pipeline_config(dataset)
+        if "coverage" in config:
```

Comment:
Would be worth adding at least a comment in here to remind ourselves that we're assuming that the coverage is specified in ISO3 values.